File: vjudge/seccom/l.cpp
#include<bits/stdc++.h>
#define ll long long
using namespace std;

int gcd(int a, int b){
	if (a == 0) return b;
	return gcd(b%a, a);
}


int main() {
	int t;cin>>t;
	int cases = 1;
	while(t--){
		int a,b,c;cin>>a>>b>>c;
		abs(a); abs(b); abs(c);
		if (c%gcd(a,b)){
			cout<<"Cases "<<cases++<<": No"<<endl;
		}else{	
			cout<<"Cases "<<cases++<<": Yes"<<endl;
		}
		
	}
	return 0;
}

File: vjudge/seccom/l.py
def gcd(a, b):
	if(a == 0):
		return b
	return gcd(b%a, a)


t = int(input())
cases = 1
for _ in range(t):
	a, b, c = map(int, input().split())
	if(c%gcd(a, b) == 0):
		print(f"Cases {cases}: Yes")
	else:
		print(f"Cases {cases}: No")
	cases+=1;	